app.config.yaml

```diff
@@ -54,3 +54,6 @@ application:
         license: Apache-2.0
         actions:
           $include: ./actions/stock/external/actions.config.yaml
+productDependencies:
```

Comment:
Didn't know about this! I just found a wiki mentioning something about it. Do you have more info about possible codes and the impact of this addition in an app builder project?